Formula/step.rb

```diff
@@ -16,8 +16,8 @@ class Step < Formula
   depends_on "go" => :build
 
   resource "certificates" do
-    url "https://github.com/smallstep/certificates/releases/download/v0.15.5/step-certificates_0.15.5.tar.gz"
-    sha256 "861690259f21d60e1491f28fd572805b1f8c3141bde651e018da06b404d7cfed"
+    url "https://github.com/smallstep/certificates/releases/download/v0.15.8/step-certificates_0.15.8.tar.gz"
```

Comment:
These are no longer in sync?

Reply:
I assume you're referring to the versions for `cli` and `certificates`? If so, then yes (although, they were already out of sync). They start off in sync at the beginning of a minor release (e.g. v0.15.0). But then we often have to make more changes `certificates` than we do to the `cli`. We'll often add new features that don't immediately require `cli` parity.

Separating these two from the same homebrew package is on our roadmap. We just need `cli` to depend on `certificates`. But we haven't gotten around to it yet. Hopefully soon.